src/core/js/models/adaptModel.js

```diff
@@ -510,6 +505,10 @@ export default class AdaptModel extends LockingModel {
       return false;
     });
 
+    const shouldSkip = model => {
```

Comment:
comment with link to issue might be useful here i.e.
// https://github.com/adaptlearning/adapt_framework/issues/3031